inst/eDISH_app/global.R

```diff
@@ -1,48 +1,19 @@
 # temporarily load deps
```

Comment:
still a wee bit confused about whether these belong here...

Reply:
Yeah, me too. I wonder if @xni7 has any advice?

Reply:
We don't need to source these functions here? They are part of the safetyGraphics R package after exported and will be loaded to the global namespace with the library call.

Also might consider including and exporting the module functions in the R package as building blocks for Shiny apps.